frame/collective/src/lib.rs

```diff
@@ -666,12 +566,17 @@ decl_module! {
 		///  - any mutations done while executing `proposal` (`P1`)
 		/// - up to 3 events
 		/// # </weight>
+		// NOTE: surprising difference between old manual weight and new generated weight:
+		// early disapproved: 37.21 vs 47.412
+		// early approved:    50.82 vs 69.466 and B * 0.008 vs 0.004 * b
+		// disapproved:       B * 0.003 vs 0 * b
+		// approved:          65.95 vs 77.033
```

Comment:
its interesting that most of the benchmark go increasing. I wonder why.

about disapproved I think it is correct to have no influence of b anymore.